frontend/SettingsPane.js

```diff
@@ -10,15 +10,16 @@
 'use strict';
 
 var BananaSlugFrontendControl = require('../plugins/BananaSlug/BananaSlugFrontendControl');
-var React = require('react');
-
+var ColorizerFrontendControl = require('../plugins/Colorizer/ColorizerFrontendControl');
 var decorate = require('./decorate');
+var React = require('react');
```

Comment:
the separation wasn't preserved w/ the addition of BananaSlug, I guess, but typically I like node_modules dependencies first, then a blank line, then local dependencies